pkg/cloudprovider/providers/openstack/openstack_test.go

```diff
@@ -557,17 +557,22 @@ func TestInstanceIDFromProviderID(t *testing.T) {
 		fail       bool
 	}{
 		{
-			providerID: "openstack://7b9cf879-7146-417c-abfd-cb4272f0c935",
+			providerID: ProviderName + "://" + "/" + "7b9cf879-7146-417c-abfd-cb4272f0c935",
```

Comment:
Yes, I agree, the second test case is origin test case:
providerID: ProviderName + "://" + "7b9cf879-7146-417c-abfd-cb4272f0c935",
is the same as "providerID: "openstack://7b9cf879-7146-417c-abfd-cb4272f0c935","